src/VisualStudio/Core/Def/Implementation/FindReferences/FindUsagesValueUsageInfoColumnDefinition.cs

```diff
@@ -33,6 +35,23 @@ public FindUsagesValueUsageInfoColumnDefinition()
         {
         }
 
+        public override bool TryGetFilterItems(ITableEntryHandle entry, out IEnumerable<string> filterItems)
+        {
+            // Determine the constituent strings for the display value in column, which should be used for applying the filter.
+            // For example, if value "Read, Write" is displayed in column for an entry, we will return "Read" and "Write" here,
+            // so filtering based on individual filter terms can be done.
+            if (IsFilterable &&
+                entry.TryGetValue(Name, out var value) &&
+                value is string displayString &&
+                !string.IsNullOrEmpty(displayString))
+            {
+                filterItems = s_displayValueToConstituentValuesMap.GetOrAdd(displayString, SplitAndTrimValue);
```

Comment:
is SplitAndTrimValue a functoin?  is this allocating a delegate each time this is called?